pkg/cloudprovider/providers/gce/gce.go

```diff
@@ -87,6 +87,11 @@ func init() {
 	cloudprovider.RegisterCloudProvider(ProviderName, func(config io.Reader) (cloudprovider.Interface, error) { return newGCECloud(config) })
 }
 
+// Raw access to the underlying GCE service, probably should only be used for e2e tests
```

Comment:
hrm, I'm doing really specific things that I don't intend to generalize.  I'd rather not define an general interface until its needed for Kubernetes proper, not just testing.